### ISSUE TITLE
Steam Furnace bug

### ISSUE THREAD

**Issue by DelosDiablo** [GregTechCE/GregTech#475]
Hello, I have found a bug with the steam furnace. Whenever I use it when it’s connected to a Small Bronze Pipe, it will only smelt one item, and then I have to break it and place it down again. It doesn’t do this when I have it placed next to a block that creates steam power. Please help!

**Comment by Archengius** [comment]
And you are sure the back side of furnace is free so steam can exit after smelting?

**Comment by DelosDiablo** [comment]
I did already try that. Let me try again (I may have a block behind it and can’t see i)

**Comment by DelosDiablo** [comment]
It was this problem (I had a block I couldn’t see behind it and I removed it.) Thank you for the help
